complete/src/main/java/com/example/android/whileinuselocation/ForegroundOnlyLocationService.kt

```diff
@@ -60,29 +60,80 @@ class ForegroundOnlyLocationService : Service() {
 
     private lateinit var notificationManager: NotificationManager
 
+    // TODO: Step 1.1, Review variables (no changes).
+    // FusedLocationProviderClient - Main class for receiving location updates.
     private lateinit var fusedLocationProviderClient: FusedLocationProviderClient
+
+    // LocationRequest - Requirements for the location updates, i.e., how often you should receive
+    // updates, the priority, etc.
     private lateinit var locationRequest: LocationRequest
+
+    // LocationCallback - Called with FusedLocationProviderClient has a new Location.
     private lateinit var locationCallback: LocationCallback
+
+    // Used only for local storage of the last known location. Usually, this would be saved to your
+    // database, but because this is a simplified sample without a full database, we only need the
+    // last location to create a Notification if the user navigates away from the app.
     private var currentLocation: Location? = null
 
     override fun onCreate() {
         Log.d(TAG, "onCreate()")
 
         notificationManager = getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
 
-        locationRequest = LocationRequest()
-        locationRequest.interval = UPDATE_INTERVAL_IN_MILLISECONDS
-        locationRequest.fastestInterval = FASTEST_UPDATE_INTERVAL_IN_MILLISECONDS
-        locationRequest.priority = LocationRequest.PRIORITY_HIGH_ACCURACY
-
+        // TODO: Step 1.2, Review the FusedLocationProviderClient.
         fusedLocationProviderClient = LocationServices.getFusedLocationProviderClient(this)
 
+        // TODO: Step 1.3, Create a LocationRequest.
+        locationRequest = LocationRequest().apply {
+            // Sets the desired interval for active location updates. This interval is inexact. You
+            // may not receive updates at all if no location sources are available, or you may
+            // receive them slower than requested. You may also receive updates faster than
```

Comment:
nit: Change "slower" and "faster" to "less frequently" and "more frequently", respectively

Applies to the comments for fastestInterval and maxWaitTime, as well